### ISSUE TITLE
Add low level bot controls

### ISSUE THREAD

**Issue by techninja** [evil-mad/robopaint#35]
N, E, S, W, and others to keep parity with RoboPaint RT. Will require lower level motor control options to allow manual movement beyond "fenced in" paint area.  See -> https://github.com/techninja/cncserver/issues/33

**Comment by docprofsky** [comment]
Would this issue be the implementation of a UI for the controls? If not then this issue does not belong here.

**Comment by techninja** [comment]
I don't think it was ever for UI, perhaps just a global key command. Also moving like this absolutely might not be directly possible with the current API
